### PR TITLE
update copyright

### LINKED ISSUE #1586: Copyright renewal
The current copyright statement is:

Copyright (c) 2013-2021, Sandia National Laboratories and pvlib python Development Team
All rights reserved.

Two problems: 2021 is past, and "Sandia National Laboratories" isn't a legal entity (neither is the "Development Team.")

Sandia's copyright office is willing to provide the copyright assertion on behalf of pvlib-python. To acknowledge that much of the code does not originate with Sandia, they advise that we change the statement to

Copyright (c) 2013-YYYY, National Technology and Engineering Solutions of Sandia, LLC and Contributors.

@pvlib/pvlib-maintainer would like your opinions here.

### PR DESCRIPTION
**Pull request by cwhanse**
- [x] Closes #1586 
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by wholmgren** [comment]
For what it's worth: https://hynek.me/til/copyright-years/

**Comment by cwhanse** [comment]
Our internal copyright document says first year only

**Review by wholmgren — APPROVED** [review]
Thanks Cliff!

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2013-2021, Sandia National Laboratories and pvlib python Development Team
+Copyright (c) 2013, National Technology & Engineering Solutions of Sandia, LLC (NTESS)and pvlib python Development Team
```
Comment (kanderso-nrel) [inline LICENSE:3]: I think a space character is needed here

**Review by kanderso-nrel — COMMENTED** [review]
Probably want to update (or ditch) the similar copyright text in `AUTHORS.md` as well

**Review by kanderso-nrel — APPROVED** [review]

**Comment by cwhanse** [comment]
@AdamRJensen what would you remove? The only text my organization requires, for the portion of pvlib written here, is

`Copyright (c) 2013, National Technology & Engineering Solutions of Sandia, LLC`

**Comment by AdamRJensen** [comment]
> @AdamRJensen what would you remove? The only text my organization requires, for the portion of pvlib written here, is
> 
> `Copyright (c) 2013, National Technology & Engineering Solutions of Sandia, LLC`

Well, with the fear of coming across as presumptuous, why isn't Sandia's copyright/authorship included in "pvlib python Development Team"? If you need to explicitly include it then I think it would send a better signal to have the "pvlib python Development Team" listed first.

**Comment by cwhanse** [comment]
@AdamRJensen I agree about the ordering.

One of the consequences of working in an organization like mine are inflexible rules about copyright and publications. According to those rules, any contribution I (or my colleagues) make is subject to copyright by our employer. It's not different at other U.S. government labs (e.g., this [comment](https://github.com/pvlib/pvlib-python/blob/7bb30ad6e9d599d10510c7b37d95d56f14e846b4/pvlib/ivtools/utils.py#L474)).

**Comment by adriesse** [comment]
So then, perhaps it should say the *portions* of pvlib are copyright Sandia?

**Comment by adriesse** [comment]
Afterthough:

Could you simply spell out in this file that Sandia is the copyright holder for those portions of pvlib-python that were originally translated from matlab in 2013, and also for those portions subsequently contributed by Sandia employees?

**Comment by kanderso-nrel** [comment]
Thanks @cwhanse for laying it out.  Speaking for myself, I'm fine with whatever you think is the right way to go here.

One thing: is the inconsistency wrt years and entity names between `AUTHORS` and `LICENSE` intentional?

```
*   Copyright (c) 2023, pvlib-python Contributors
*   Copyright (c) 2014, pvlib python Development Team
*   Copyright (c) 2013, Sandia National Laboratories
```
versus
```
Copyright (c) 2015 pvlib python Contributors
Copyright (c) 2014 PVLIB python Developers
Copyright (c) 2013 Sandia National Laboratories
```

**Comment by cwhanse** [comment]
> One thing: is the inconsistency wrt years and entity names between AUTHORS and LICENSE intentional?

Not intentional, I'll fix that.

**Comment by adriesse** [comment]
Thanks very much @cwhanse.  I'm going to book mark that for future reference!

I was trying to suggest something that might alleviate @AdamRJensen's concern about perception, rather than object to anything fundamental.  It seems to me with the three copyright statements in reverse chronological order the perception issue is probably resolved.

**Comment by kanderso-nrel** [comment]
Judging by the thumbs-up reactions and absence of continued objection, I'll assume that we're all on board with the PR as it currently stands and merge it.  Thanks @cwhanse and all discussion participants

**Pull request merged**